worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/ArgumentsManager.java

```diff
@@ -191,7 +276,9 @@ else if (type.equals("cd")){
 
         final ResolvedSetting resolvedSetting;
         try {
-            resolvedSetting = settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
+            resolvedSetting = shouldForceCacheRefresh(name, scopes, priorities, settingsServiceLastUpdateTimeMillisOpt)
+                ? forceCacheRefreshSettingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities))
+                : settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
```

Comment:
You can eliminate the duplication by selecting the appropriate settings api:
```suggestion
            resolvedSetting = (shouldForceCacheRefresh(name, scopes, priorities, settingsServiceLastUpdateTimeMillisOpt)
                ? forceCacheRefreshSettingsApi : settingsApi)
                .getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
```